spi-fly/spi-fly-itests/test.bndrun

```diff
@@ -51,16 +51,17 @@
     bnd.identity;id='junit-platform-launcher'
 
 -runbundles: \
-	assertj-core;version='[3.22.0,3.22.1)',\
+	assertj-core;version='[3.23.1,3.23.2)',\
 	junit-jupiter-api;version='[5.8.2,5.8.3)',\
 	junit-jupiter-engine;version='[5.8.2,5.8.3)',\
 	junit-jupiter-params;version='[5.8.2,5.8.3)',\
 	junit-platform-commons;version='[1.8.2,1.8.3)',\
 	junit-platform-engine;version='[1.8.2,1.8.3)',\
 	junit-platform-launcher;version='[1.8.2,1.8.3)',\
-	org.apache.aries.spifly.dynamic.framework.extension;version='[1.3.5,1.3.6)',\
-	org.apache.aries.spifly.examples.spi.bundle;version='[1.0.3,1.0.4)',\
-	org.apache.aries.spifly.itests;version='[1.3.5,1.3.6)',\
+	net.bytebuddy.byte-buddy;version='[1.12.10,1.12.11)',\
+	org.apache.aries.spifly.dynamic.framework.extension;version='[1.4.0,1.4.1)',\
+	org.apache.aries.spifly.examples.spi.bundle;version='[1.0.5,1.0.6)',\
+	org.apache.aries.spifly.itests;version='[1.4.0,1.4.1)',\
```

Comment:
These changes took place automatically when I performed a local build, so I think they might have been missed in a previous update.